test/Microsoft.TemplateEngine.Core.UnitTests/ConditionalTests.CStyleEvaluator.cs

```diff
@@ -1,11 +1,24 @@
 ﻿using System.Collections.Generic;
+using System.Globalization;
 using System.IO;
 using System.Text;
 using Microsoft.TemplateEngine.Core.Contracts;
 using Xunit;
 
 namespace Microsoft.TemplateEngine.Core.UnitTests
 {
+
+    public sealed class SkipOnNonEnUsLocale : FactAttribute
+    {
+        public SkipOnNonEnUsLocale(string displayName, string message) {
+            DisplayName = displayName;
+            if (!CultureInfo.InstalledUICulture.Name.Equals("en-US"))
```

Comment:
This should be `CultureInfo.CurrentCulture`. Example: On English Windows I can set "Region" to Russia which will affect .NET formatting/parsing and expose the bug but `InstalledUICulture` will still be `en-US`.

Reply:
I see, fixed this. Thanks!